### ISSUE TITLE
Not working with inputs outside of the form

### ISSUE THREAD

**Issue by Nuranto** [proengsoft/laravel-jsvalidation#377]
Here is a fix : https://github.com/jquery-validation/jquery-validation/pull/2261

Also, an update of the js lib would be great. (I tried, but it seems it doesn't work without some compatibility fixes here). But less imporant.

**Comment by bytestream** [comment]
Latest is on master branch

**Comment by Nuranto** [comment]
@bytestream No, it does not.
I had to add this change : 
```
-			return $( this.currentForm )
-			.find( "input, select, textarea, [contenteditable]" )
+			return return $.merge( $( this.currentForm ).find( "input, select, textarea, [contenteditable]" ),  
+                                        $( this.currentForm.elements ).filter( "input, select, textarea, [contenteditable]" ) 
+                        )

```

**Comment by bytestream** [comment]
> Also, an update of the js lib would be great. 

I was replying to the above. The latest version of this library is using the latest version of jquery-validation.

> Here is a fix : jquery-validation/jquery-validation#2261

Validating elements outside the form is not supported. You'd have to apply pressure to jquery-validation or apply the changes yourself.

I don't intend to maintain our own fork of jquery-validation in this library.
